docs/UsersGuide/source/AQM.rst

```diff
@@ -170,3 +136,127 @@ Users may check experiment status from the experiment directory with either of t
    # Check the experiment status and relaunch the workflow (for manual jobs)
    ./launch_FV3LAM_wflow.sh; tail -n 40 log.launch_FV3LAM_wflow
 
+
+WE2E Test for AQM
+=======================
+
+Build the app for AQM:
+
+.. code-clock:: console
+
+  ./devbuild.sh -p=hera -a=ATMAQ
+
+
+Add the WE2E test for AQM to the list file:
+
+.. code-block:: console
+
+   echo "custom_ESGgrid" > my_tests.txt
+   echo "aqm_grid_AQM_NA13km_suite_GFS_v16" >> my_tests.txt
+
+
+Run the WE2E test:
+
+.. code-block:: console
+
+   $ ./run_WE2E_tests.py -t my_tests.txt -m hera -a gsd-fv3 -q
+
+
+
+Additional Tasks for AQM
+===============================
+
+Structure of SRW-AQM
+-------------------------
+
+The flowchart of the non-DA (data assimilation) SRW-AQM (Air Quality Modeling) is illustrated in :numref: `Figure %s <FlowProcAQM>. Compared to the non-coupled (ATM stand-alone) FV3-LAM, SRW-AQM has additional tasks for pre- and post-processing. For pre-processing, multiple emission data such as NEXUS, fire, and point-source emission are retrieved or created for air quality modeling. Moreover, the chemical initial conditions (ICs) are extracted from the restart files of the previous cycle and added to the existing IC files. The chemical lateral boundary conditions (LBCs) and the GEFS aerosol data are also adeded to the existing LBC files. For post-processing, air quality forecast products for O3 and PM2.5 are generated and the bias-correction technique is applied to improve the accuracy of the results.
+
+.. _FlowProcAQM:
+
+.. figure:: _static/SRW-AQM_workflow.png
+      :alt: Flowchart describing the tasks of SRW-AQM.
+
+   *Workflow structure of SRW-AQM (non-DA)*
+
+
+
+Pre-processing Tasks of SRW-AQM
+------------------------------------
+
+The pre-processing tasks for air quality modeling (AQM) are shown in :numref:`Table %s <TasksPrepAQM>.
```

Comment:
Need to add a ` to the end of :numref:Table %s TasksPrepAQM
```suggestion
The pre-processing tasks for air quality modeling (AQM) are shown in :numref:`Table %s <TasksPrepAQM>`.
```

Reply:
Thanks, @chan-hoo!  The correct table is now in the user's guide.  Resolving now.